gradle/plugins/common/src/main/kotlin/junitbuild.java-nullability-conventions.gradle.kts

```diff
@@ -0,0 +1,32 @@
+import junitbuild.extensions.dependencyFromLibs
+import junitbuild.extensions.javaModuleName
+import net.ltgt.gradle.errorprone.errorprone
+import net.ltgt.gradle.nullaway.nullaway
+
+plugins {
+	`java-library`
+	id("net.ltgt.errorprone")
+	id("net.ltgt.nullaway")
+}
+
+dependencies {
+	errorprone(dependencyFromLibs("errorProne-core"))
+	errorprone(dependencyFromLibs("nullaway"))
+}
+
+nullaway {
+	annotatedPackages.add(javaModuleName)
+}
+
+tasks.withType<JavaCompile>().configureEach {
+	options.errorprone {
+		disableAllChecks = true
```

Comment:
We can later (after this PR) enable additional checks provided by Error Prone but I didn't want to mix those changes into this already large set of changes.